app/controller/sdl/Abstract/Controller.js

```diff
@@ -1549,5 +1549,20 @@ SDL.SDLController = Em.Object.extend(
       }
       return title;
     },
+
+    imageModeChanged: function() { 
+      if (SDL.SDLController.model) {
```

Comment:
@YarikMamykin try to make if-return for a corner cases in order to avoid large nested if's
```suggestion
      if (!SDL.SDLController.model) {
        return;
      }
```

Reply:
@AKalinich-Luxoft changed in https://github.com/LuxoftSDL/sdl_hmi/pull/10/commits/800bde30ba931dbfec1d5e9fc1939c1fe7836909